src/modules/UsersPluginKerberos.pm

```diff
@@ -258,11 +268,23 @@ sub Write {
     }
 
     if ( defined $input ) {
-      my $pid = open3(\*IN, \*OUT, \*ERR, "$command")
-      or do {
-        $error = __("Cannot execute kadmin.local.");
-        return YaST::YCP::Boolean (0);
+      my $failure = 0;
+      my $pid = open3(\*IN, \*OUT, \*ERR, "$command") or do {
+	$failure = 1;
       };
+
+      if ( $failure ) {
+	$failure = 0;
+	$pid = open3(\*IN, \*OUT, \*ERR, "$old_command") or do {
+          $failure = 1;
+	};
+      }
+
+      if ( $failure ) {
+	$error = __("Cannot execute kadmin.local.");
+        return YaST::YCP::Boolean (0);
```

Comment:
in general this solution looks quite hacky for me. Why not during init just check by e.g. file stat if file exists?

Reply:
I agree with Josef. Why not simply using `-x $command` to determine which command to use, then run it.